x/bsonx/bsoncore/bsoncore.go

```diff
@@ -198,10 +197,10 @@ func ReadString(src []byte) (string, []byte, bool) {
 // AppendDocumentStart reserves a document's length and returns the index where the length begins.
 // This index can later be used to write the length of the document.
 //
-// TODO(skriptble): We really need AppendDocumentStart and AppendDocumentEnd.
-// AppendDocumentStart would handle calling ReserveLength and providing the index of the start of
-// the document. AppendDocumentEnd would handle taking that start index, adding the null byte,
-// calculating the length, and filling in the length at the start of the document.
+// TODO(skriptble): We really need AppendDocumentStart and AppendDocumentEnd.  AppendDocumentStart would handle calling
+// TODO ReserveLength and providing the index of the start of the document. AppendDocumentEnd would handle taking that
+// TODO start index, adding the null byte, calculating the length, and filling in the length at the start of the
+// TODO document.
 func AppendDocumentStart(dst []byte) (index int32, b []byte) { return ReserveLength(dst) }
```

Comment:
Same as above, move the TODO comment block to the first line of the `AppendDocumentStart` function.